package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "protocol",
+  "version": "1.0.0",
+  "dependencies": {},
+  "devDependencies": {
+    "@bazel/typescript": "2.0.3",
+    "typescript": "3.7.5",
+    "grpc-tools": "^1.9.1",
+    "grpc_tools_node_protoc_ts": "^5.0.1"
+  }
+}
```

Comment:
Node dependencies in order to compile typescript

Reply:
Do we need both `package.json` files?